collated/main/cctdaniel.md

```diff
@@ -0,0 +1,452 @@
+ * A Enumeration class that consists of all possible FontSize
+ * Unit in the panel.
+ */
+public enum FontSizeUnit {
+    FONT_SIZE_XSMALL_UNIT, FONT_SIZE_SMALL_UNIT, FONT_SIZE_NORMAL_UNIT, FONT_SIZE_LARGE_UNIT, FONT_SIZE_XLARGE_UNIT;
+
+    private static FontSizeUnit currentFontSizeUnit = FONT_SIZE_NORMAL_UNIT;
+
+    /** Get current FontSize unit */
+    public static FontSizeUnit getCurrentFontSizeUnit() {
+        return currentFontSizeUnit;
+    }
+
+    /** Reset FontSize unit in the panel with the new FontSizeUnit */
+    public static void setCurrentFontSizeUnit(FontSizeUnit unit) {
+        currentFontSizeUnit = unit;
+    }
+}
+```
+###### \java\seedu\address\model\ThemeUnit.java
+``` java
+/**
+ * A Enumeration class that consists of all possible Theme
+ * Unit in the panel.
+ */
+public enum ThemeUnit {
+    THEME_LIGHT_UNIT, THEME_DARK_UNIT;
+
+    private static ThemeUnit currentThemeUnit = THEME_LIGHT_UNIT;
+
+    /** Get current Theme unit */
+    public static ThemeUnit getCurrentThemeUnit() {
+        return currentThemeUnit;
+    }
+
+    /** Reset Theme unit in the panel with the new ThemeUnit */
+    public static void setCurrentThemeUnit(ThemeUnit unit) {
+        currentThemeUnit = unit;
+    }
+}
+```
+###### \java\seedu\address\ui\CommandBox.java
+``` java
+    @Subscribe
+    private void handleChangeFontSizeEvent(ChangeFontSizeEvent event) {
+        setFontSize(event.message);
+    }
+```
+###### \java\seedu\address\ui\CommandBox.java
+``` java
+    /**
+     * Sets the command box style to user preferred font size.
+     */
+    private void setFontSize(String userPref) {
+        switch (userPref) {
+        case FONT_SIZE_XSMALL:
+            commandTextField.setStyle("-fx-font-size: x-small;");
+            userPrefFontSize = "-fx-font-size: x-small;";
+            fontIndex = 1;
+            break;
+
+        case FONT_SIZE_SMALL:
+            commandTextField.setStyle("-fx-font-size: small;");
+            userPrefFontSize = "-fx-font-size: small;";
+            fontIndex = 2;
+            break;
+
+        case FONT_SIZE_NORMAL:
+            commandTextField.setStyle("-fx-font-size: normal;");
+            userPrefFontSize = "-fx-font-size: normal;";
+            fontIndex = 3;
+            break;
+
+        case FONT_SIZE_LARGE:
+            commandTextField.setStyle("-fx-font-size: x-large;");
+            userPrefFontSize = "-fx-font-size: x-large;";
+            fontIndex = 4;
+            break;
+
+        case FONT_SIZE_XLARGE:
+            commandTextField.setStyle("-fx-font-size: xx-large;");
+            userPrefFontSize = "-fx-font-size: xx-large;";
+            fontIndex = 5;
+            break;
+
+        default:
+            break;
+        }
+    }
+
+    /**
+     * Initializes the history snapshot.
+     */
+    private void initHistory() {
+        historySnapshot = logic.getHistorySnapshot();
+        // add an empty string to represent the most-recent end of historySnapshot, to be shown to
```

Comment:
Avoid using comments for explaining code. Code should be self-explanatory. If you still want to add this information then add to the Developer Guide.